### ISSUE TITLE
Research Beyond Academia

### ISSUE THREAD

**Issue by Alrubin** [studentreps/vademecum#7]
We should specify better which are the activities of the Valorization office. In particular

- the convention with Coursera
- the fact that the Master in HPC is paid by SISSA to former students
- the convention for the 24 CFU for teaching with UniTs

I further suggest to rename this section and move it somewhere else.

**Comment by matteosecli** [comment]
Are we totally sure about the second point? I know that there have been some issues with that, recently.

Ideas about the relocation?

**Comment by Alrubin** [comment]
I'm sure it's paid. Maybe we can leave there that section but renamed "Career Beyond Academia". What do you think ?

**Comment by matteosecli** [comment]
"Career Beyond Academia" LGTM 👍

I'll try to get my hands on the actual regulation for the MPHC, so that at least we have a reference on that.

**Comment by matteosecli** [comment]
Have found the following in the [2020/2021 tender](https://www.sissa.it/sites/default/files/Decreto%20e%20bando%20MHPC%202020-2021.pdf):

> **Support for SISSA PhD students and former SISSA PhD students**
> SISSA waves the fee of MHPC to its PhD students who are enrolled on one of its PhD courses on the date of the official start of MHPC classes, and to former students who have obtained their doctorate no more than 12 months before the start date of the lessons and that are currently unemployed, for a maximum of _six exemptions_. The exemption is granted to the first three PhD students and to the first three former PhD students that are admitted to the Master, according to the ranking of the admission.
> Should the number of admitted students or former students be less than three, the exemptions not enjoyed by one of the two categories will be transferred to the other. Beneficiaries of the exemption are required to complete the training course, under penalty of paying € 1.000,00.-. to SISSA, as a contribution to the training costs.
> If the beneficiary of an exemption is awarded a scholarship that also includes a contribution for the fees (see next section), the benefit of the exemption is withdrawn and the student is required to pay the balance of the entire fee for participation in MHPC.
> SISSA PhD students are encouraged to enroll in MHPC during their second to fourth year of the PhD. If they result among the admitted MHPC students, they must obtain a _nulla osta_ from their advisors and from the “collegio dei docenti” of their PhD. SISSA PhD scholarships are **not** compatible with other scholarships, and cannot be cumulated.
> 
> **External scholarships**
> Other research institutes as well as private companies and industries may offer scholarships to cover the fee of the master and/or partial/full living expenses. All of these scholarships will be advertised on the website www.mhpc.it, and will have their own deadlines and conditions. These are independent with respect to the MHPC application and deadline, and application/admission to these scholarships **will not** imply application/admission to MHPC, which must be actuated separately. The only exception to this rule is given by winners of ICTP scholarships for developing countries, who are granted admission to MHPC.

So, basically:
- Max 6 scholarships, equally shared between current PhD students and former PhD students — unless there are so few people that the unused scholarships in one group can be transferred to the other one.
- If you're a former PhD student, you have to be within 1 year from graduation and be unemployed to qualify.
- If you find a company that pays the fee for you, then SISSA's scholarship will be used for someone else.

Have I forgotten something?

Btw these scholarships were **not** there in the [2019/2020 tender](https://www.sissa.it/sites/default/files/Bando%20MHPC_2019_2020.pdf), it seems to be a new thing (I guess after the storm that mounted after that specific tender). Do we know how long will it last, or was it a _una tantum_ provision?

**Comment by Alrubin** [comment]
Da quel che so io (che come CdA ho ratificato quell'accordo), è una cosa da qui a sempre. Per scrupolo possiamo mettere che è una cosa da verificare di volta in volta sul bando.  che dici ?

**Comment by matteosecli** [comment]
Yep, I think we can point out this possibility but tell people to be sure to check the tender anyway!

**Comment by fcaleffi** [comment]
Given the official presentation of the MHPC convention 06/07/2021, I think that it can be classified as an official thing now. @Alrubin, do you want to take care of rewriting the "Research Beyond Academia" section? Otherwise, I'll do that as soon as possible.

**Comment by Alrubin** [comment]
I would, but I have already too many things to do with the thesis in this period. I would appreciate it so much if you did it.

**Comment by fcaleffi** [comment]
No problem, I'll go through with it! :+1:

**Comment by fcaleffi** [comment]
Regarding the proposal of moving the "Research Beyond Academia" elsewhere, what about upgrading it to a whole chapter or at least shifting it under the dome of the "Extracurricular Activities" section? And how would you rename the section? After this round of suggestions, I'll start writing down something.

**Comment by Alrubin** [comment]
I like the idea of creating a new Chapter called e.g. "A look after your PhD" where the topics are 
- SISSA Alumny Society
- Suggestions to Find a Post-Doc
- Valorization Office and their initiatives
- The Coursera Online Platform
- The Master in High Performance Computing and in Science Communication
- convention with UniTs for the 24 CFU for teaching 
- SISSA contributions to fund a start-up
Note that in this case we have to rename chapter 3 (Research, Academic Life and Beyond  ------> Research and Academic Life) and Part I (Life at SISSA -----> Life at and after SISSA).

**Comment by fcaleffi** [comment]
Short update! I've taken care of almost all the sections listed by @Alrubin (OVERLEAF ONLY), except for:

- SISSA Contributions to Fund a Start-Up Company
- Suggestions to Find a Post-Doc

I know a little about these last points, unfortunately. Also, I think that probably it would be useful to ask past colleagues for useful tips to be put in the second section.
